apps/gitness/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react-swc'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  server: {
+    port: 5137, // UI dev server port
+    proxy: {
+      '/repos': {
```

Comment:
we need to fix this with a proper `/api/code` prefix

Reply:
Fixed it, pls check

Reply:
Done, as per discussion.